pkg/labels/filter.go

```diff
@@ -85,10 +85,10 @@ func (p LabelPrefix) matches(l *Label) (bool, int) {
 // parseLabelPrefix returns a LabelPrefix created from the string label parameter.
 func parseLabelPrefix(label string) (*LabelPrefix, error) {
 	labelPrefix := LabelPrefix{}
-	t := strings.SplitN(label, ":", 2)
-	if len(t) > 1 {
-		labelPrefix.Source = t[0]
-		labelPrefix.Prefix = t[1]
+	i := strings.IndexByte(label, ':')
+	if i >= 0 {
```

Comment:
`IndexByte()` returns non-negative index when the byte is found. `SplitN()` returns 2 if the delimiter is found. This test was `> 1` before, so the corresponding test is `>= 0`. Are you saying that an empty source like ":prefix" should be parsed as `{Source="", Prefix=":prefix"}` instead of `{Source="", Prefix="prefix"}`? If so this would be a bug fix that I'd separate to a commit of it's own.

Reply:
> `IndexByte()` returns non-negative index when the byte is found. `SplitN()` returns 2 if the delimiter is found. 

I forgot that.

>This test was `> 1` before, so the corresponding test is `>= 0`. Are you saying that an empty source like ":prefix" should be parsed as `{Source="", Prefix=":prefix"}` instead of `{Source="", Prefix="prefix"}`? 

No